pkg/cmd/step/create/step_create_install_secrets.go

```diff
@@ -0,0 +1,142 @@
+package create
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/jenkins-x/jx/pkg/cmd/helper"
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/helm"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/engine"
+)
+
+const (
+	// SecretsTemplateFileName contains the template used to generate the secrets.yaml file which can be safely checked into git
+	SecretsTemplateFileName = "template-secrets.yaml"
+
+	// SecretsParametersFileName the secret parameter files which are never checked into git and may come from, say, Vault
+	SecretsParametersFileName = "secrets-parameters.yaml"
+)
+
+var (
+	createInstallSecretsLong = templates.LongDesc(`
+		Creates or updates the install/upgrade secrets.yaml file from a template you can safely check into git: 'template-secrets.yaml' and a local secret parameters file: 'secrets-parameters.yaml'
+
```

Comment:
It would be great if we could be consistent with notation and with the secrets sources.